integration/uma/src/test/resources/simplelogger.properties

```diff
@@ -1 +1 @@
-org.slf4j.simpleLogger.log.org.eclipse.jetty=warn
+org.slf4j.simpleLogger.log.org.eclipse.jetty=INFO
```

Comment:
Is there a reason for this change?

Reply:
This is useful for debugging, but it would (likely) be best to keep it at `warn` level